src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
+
+# Installing Ceph
+
+Here we present one way to install a rudimentary Ceph cluster with Juju backed
+by an AWS cloud. See [Ceph documentation][upstream-ceph-documentation] to find
+out more about Ceph.
```

Comment:
This starts to feel a bit tutorial-ish and regardless I think that it's worth asking the storage expert if there's anything just plain wrong/bad practice that's setup in a doc like this.